src/addons/outlook/src/taskpane/taskpane.css

```
@@ -0,0 +1,147 @@
+    margin: 20px 0;
+}
+
+/* ── ProConnect ── */
+.proconnect-sr-only {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.9.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)